packages/p2p/src/components/modal-manager/modals/share-my-ads-modal/share-my-ads-card/share-my-ads-card.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { QRCode } from 'react-qrcode-logo';
+import { Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { Localize } from 'Components/i18next';
+import { base64_images } from 'Constants/base64-images';
+import { TAdvertProps } from 'Types';
+
+type TShareMyAdsCardProps = {
+    advert: Partial<TAdvertProps>;
+    advert_url: string;
+    divRef: React.MutableRefObject<HTMLDivElement> | React.MutableRefObject<null>;
+};
+
+const ShareMyAdsCard = ({ advert, advert_url, divRef }: TShareMyAdsCardProps) => {
+    const { account_currency, id, max_order_amount_limit_display, min_order_amount_limit_display, rate_display, type } =
+        advert;
+
+    const options = {
+        enableCORS: true,
+        size: isMobile() ? 120 : 150,
+        removeQrCodeBehindLogo: true,
+        logoPadding: 4,
+        logoImage: base64_images.dp2p_logo,
+        logoWidth: isMobile() ? 30 : 40,
+        logoHeight: isMobile() ? 30 : 40,
+        logoOpacity: 1,
```

Comment:
based on the documentation the default value for logoOpacity is already set to 1 so we can remove this